README.md

```diff
@@ -99,44 +90,52 @@ Depending on your configuration, it may be necessary to start additional service
    - See the Quick Launch Kafka section.
  - `inf-triton`
    - Requires a running Triton server
-   - See the launching Triton section.
+   - See the Launching Triton Server section.
 
-### Quick-Launch Kafka Cluster
+### Quick Launch Kafka Cluster
```

Comment:
All this Kafka docker-compose stuff is a little icky.

Reply:
Define "icky"